app/js/components/LabTrendsPage.jsx

```diff
@@ -75,8 +83,32 @@ export class LabTrendsPage extends PureComponent {
     const { location: { state } } = history;
     const { defaultPageSize } = this.state;
     const fields = ["SAMPLE DATE", "RESULT", "NORMAL RANGE"];
+
+
+    if (!state) {
+      return <Redirect to="/labresults" />;
+    }
+
+    const isNumeric = state.datatype.display === "Numeric";
+
+    const resultTableClassName = cn({
+      'lab-result-trend-list': true,
+      full: !isNumeric,
+    });
+
+    const chartClassName = cn({
+      'lab-results-trend-chart': true,
+      hide: !isNumeric,
+    });
+
     this.setState({ defaultPageSize: calculateTableRows(results.length) });
 
+    const formatChartData = data => data.map(item => ({
```

Comment:
Is it guaranteed that the data would always be sorted by date? If not this would be a good place to sort the data by date. because it seems reChart doesn't organize the data, it basically renders what it gets as is.

Reply:
Agreed, we should sort by date to be safe